### ISSUE TITLE
When fix patch is applied to my dataset, the dataset is unbalanced. Why does the training set perform well, but the loss of verification set first decreases and then keeps rising?

### ISSUE THREAD

**Issue by a571-op** [google-research/fixmatch#51]
![image](https://user-images.githubusercontent.com/76274232/102835427-a65b0000-4431-11eb-8544-934fae55a45c.png)

![image](https://user-images.githubusercontent.com/76274232/102835446-b672df80-4431-11eb-84b4-c29ababfddfc.png)
